zkevm-circuits/src/copy_circuit.rs

```diff
@@ -373,7 +373,7 @@ impl<F: Field> CopyCircuit<F> {
             || "assign copy table",
             |mut region| {
                 let mut offset = 0;
-                for copy_event in block.copy_events.values() {
+                for copy_event in &block.copy_events {
```

Comment:
```suggestion
                for copy_event in block.copy_events.iter() {
```

This adds consistency to the rest of the changes. Nitpick